nft_ingester/src/processors/accounts_processor.rs

```diff
@@ -125,6 +127,7 @@ impl<T: UnprocessedAccountsGetter> AccountsProcessor<T> {
             MplCoreFeeProcessor::build(postgre_client, metrics.clone(), rpc_client, join_set)
                 .await?;
         core_fees_processor.update_rent(rx).await;
+        let processor_name = Uuid::new_v4().to_string();
```

Comment:
I'd suggest we use the same uuid for redis and for the processor. Either inject it into both or at least in one of those